docker/docker.go

```diff
@@ -111,7 +111,7 @@ func main() {
 			*flTls = true
 			cert, err := tls.LoadX509KeyPair(*flCert, *flKey)
 			if err != nil {
-				log.Fatalf("Couldn't load X509 key pair: %s. Key encrypted?", err)
+				log.Fatalf("Couldn't load X509 key pair: %s. Key encrypted", err)
```

Comment:
I am so much agreed with u. The log here is confsing, shall we modify it to be clearly express what we want.